scripts/build.py

```diff
@@ -589,7 +591,7 @@ def build_files(source, target, file_hashes):
     for root, dirs, files in os.walk(os.path.join(os.getcwd(), source)):
         tasks = []
         for directory in dirs:
-            print 'Processing %s' % os.path.join(root, directory)
+            print 'Building %s' % os.path.join(root, directory)
```

Comment:
Thanks for these renames -- "processing" is super vague.

Reply:
Yep I was dealing with 1k lines of "Processing" so I had to do this.